_2017_pages/code_of_conduct.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: 2017_default
```

Comment:
Souldn't it be "2020_default"?

Reply:
its either 2016 or 2017, it has to be the name of an existing file